src/plot_splines.jl

```diff
@@ -15,13 +15,13 @@ Dashed lines indicate spline knots.
     `Figure`, `GridLayout`, or `GridPosition` to draw the plot.
 - `m::UnfoldModel`\\
     UnfoldModel with splines.
-- `spline_axis::NamedTuple = (;)`\\
+- `spline_kwargs::NamedTuple = (;)`\\
```

Comment:
why kwargs and not axis? I think this is a bit ambiguous because it is not clear whether they are put into the axis (as it is the case afaik) or to the visual (the lines kwargs)

Reply:
okay i will take spline_axis